include/sys/select.h

```diff
@@ -71,11 +71,11 @@
 /* Standard helper macros */
 
 #define FD_CLR(fd,set) \
-  ((((fd_set*)(set))->arr)[_FD_NDX(fd)] &= ~(1 << _FD_BIT(fd)))
+  ((((fd_set*)(set))->arr)[_FD_NDX(fd)] &= ~(1ul << _FD_BIT(fd)))
```

Comment:
```suggestion
  ((((fd_set*)(set))->arr)[_FD_NDX(fd)] &= ~(UINT32_C(1) << _FD_BIT(fd)))
```
The suffix for the integer type is architecture-dependent.
Since this is a common header, it is recommended to use the macros from `inttypes.h` for applying the proper suffix: